sql/commands/sql-create-connection.mdx

```diff
@@ -1,44 +1,103 @@
 ---
 title: "CREATE CONNECTION"
-description: "Use the `CREATE CONNECTION` command to create an AWS PrivateLink connection for a Kafka connector."
+description: "Use the `CREATE CONNECTION` command to reuse connector properties."
 ---
 
-This is necessary in order to be able to consume messages from a Kafka service located in a different VPC from the RisingWave cluster in the cloud.
+The `CREATE CONNECTION` command creates a reusable connection configuration that can be referenced when creating sources, sinks, or tables. Currently supported connection types are Kafka and schema registry.
 
 ## Syntax
 
 ```sql
 CREATE CONNECTION [ IF NOT EXISTS ] connection_name
 WITH (
-    connection_parameter = 'value'
+    type = '<connector_type>',
+    connection_parameter = SECRET `<secret_name>`,
+    ...
 );
 ```
 
-## Parameters
+Connection parameters can reference secrets using the `SECRET` keyword. This allows sensitive information to be stored securely and referenced in the connection configuration.
```

Comment:
additionally, alter secret can be automatically applied, no need to alter connection.